app/src/main/java/org/mozilla/fenix/components/TrackingProtectionPolicyFactory.kt

```diff
@@ -106,10 +107,16 @@ class TrackingProtectionPolicyFactory(
 }
 
 @VisibleForTesting
-internal fun TrackingProtectionPolicyForSessionTypes.adaptPolicyToChannel(): TrackingProtectionPolicyForSessionTypes {
+internal fun TrackingProtectionPolicy.applyTCPIfNeeded(settings: Settings): TrackingProtectionPolicyForSessionTypes {
```

Comment:
Repurposing `adaptPolicyToChannel`, as it wasn't use anymore see https://github.com/mozilla-mobile/fenix/pull/23056